docs/source/user-guide/supported_models.rst

```diff
@@ -0,0 +1,56 @@
+.. _supported_models:
+
+====================
+Supported Models
+====================
+
+Deepchecks supports a minmal version of the sklearn api.
```

Comment:
I would rephrase it a bit different:

Similar to scikit-learn API, a model will be compatible with deepchecks' checks if it implements some minimal functionality. Below are the minimal requirements for each task type

Reply:
+1